docs/quick-start.md

```diff
@@ -50,7 +50,7 @@ As the table illustrates, you need to create 2 subdomains (`matrix.example.com`
 
 It might take some time for the DNS records to propagate after creation.
 
-**Note**: if you are using Cloudflare DNS, make sure to disable the proxy and set all records to "DNS only"
+**💡 Note**: if you are using Cloudflare DNS, make sure to disable the proxy and set all records to "DNS only"
```

Comment:
Off-topic (as a memo): shall we create a emoji guide for consistency on a style guide or something?

- Warning emoji with `**Warning**:`
- Light bulb emoji with `**Note(s)**:`

We might need some time for consideration.